<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 PDAC Subtypes | PRECODE ESR Bioinformatics Course</title>
  <meta name="description" content="Basic introduction to RNAseq analysis using R/Bioconductor" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 PDAC Subtypes | PRECODE ESR Bioinformatics Course" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Basic introduction to RNAseq analysis using R/Bioconductor" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 PDAC Subtypes | PRECODE ESR Bioinformatics Course" />
  
  <meta name="twitter:description" content="Basic introduction to RNAseq analysis using R/Bioconductor" />
  

<meta name="author" content="Peter J. Bailey" />


<meta name="date" content="2021-02-20" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="rnaseq-analysis-deseq2.html"/>

<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PRECODE ESR Bioinformatics Course</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Prerequisites</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#installing-r-and-rstudio"><i class="fa fa-check"></i><b>1.1</b> Installing R and RStudio</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#installing-r-and-bioconductor-packages"><i class="fa fa-check"></i><b>1.2</b> Installing R and Bioconductor packages</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#loading-r-and-bioconductor-packages"><i class="fa fa-check"></i><b>1.3</b> Loading R and Bioconductor packages</a></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#learning_resources"><i class="fa fa-check"></i><b>1.4</b> Learning resources</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>2</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1" data-path="intro.html"><a href="intro.html#course-outline"><i class="fa fa-check"></i><b>2.1</b> Course outline</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="intro.html"><a href="intro.html#part1"><i class="fa fa-check"></i><b>2.1.1</b> PART 1</a>
<ul>
<li class="chapter" data-level="2.1.1.1" data-path="intro.html"><a href="intro.html#practicals"><i class="fa fa-check"></i><b>2.1.1.1</b> Practicals</a></li>
<li class="chapter" data-level="2.1.1.2" data-path="intro.html"><a href="intro.html#assessment"><i class="fa fa-check"></i><b>2.1.1.2</b> Assessment</a></li>
</ul></li>
<li class="chapter" data-level="2.1.2" data-path="intro.html"><a href="intro.html#part2"><i class="fa fa-check"></i><b>2.1.2</b> PART 2</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html"><i class="fa fa-check"></i><b>3</b> R Programming language primer</a>
<ul>
<li class="chapter" data-level="3.1" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#vectors"><i class="fa fa-check"></i><b>3.1</b> Vectors</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#vectors-in-action"><i class="fa fa-check"></i><b>3.1.1</b> Vectors in action</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#matrix"><i class="fa fa-check"></i><b>3.2</b> Matrix</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#matrices-in-action"><i class="fa fa-check"></i><b>3.2.1</b> Matrices in action</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#data-frame"><i class="fa fa-check"></i><b>3.3</b> Data Frame</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#data-frames-in-action"><i class="fa fa-check"></i><b>3.3.1</b> Data Frames in action</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#factors"><i class="fa fa-check"></i><b>3.4</b> Factors</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#factors-in-action"><i class="fa fa-check"></i><b>3.4.1</b> Factors in action</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#list"><i class="fa fa-check"></i><b>3.5</b> List</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#lists-in-action"><i class="fa fa-check"></i><b>3.5.1</b> Lists in action</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#array"><i class="fa fa-check"></i><b>3.6</b> Array</a></li>
<li class="chapter" data-level="3.7" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#creating-functions"><i class="fa fa-check"></i><b>3.7</b> Creating functions</a></li>
<li class="chapter" data-level="3.8" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#loading-data"><i class="fa fa-check"></i><b>3.8</b> Loading Data</a></li>
<li class="chapter" data-level="3.9" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#saving-data"><i class="fa fa-check"></i><b>3.9</b> Saving Data</a></li>
<li class="chapter" data-level="3.10" data-path="r-programming-language-primer.html"><a href="r-programming-language-primer.html#saving-plots"><i class="fa fa-check"></i><b>3.10</b> Saving plots</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html"><i class="fa fa-check"></i><b>4</b> Descriptive statistics and plotting</a>
<ul>
<li class="chapter" data-level="4.1" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#the-cancer-gene-atlas-tcga---pancreatic-ductal-adenocarcinoma-data-set"><i class="fa fa-check"></i><b>4.1</b> The Cancer Gene Atlas (TCGA) - Pancreatic Ductal Adenocarcinoma Data Set</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#gene-expression-data"><i class="fa fa-check"></i><b>4.1.1</b> Gene expression data</a></li>
<li class="chapter" data-level="4.1.2" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#clinical-data"><i class="fa fa-check"></i><b>4.1.2</b> Clinical data</a></li>
<li class="chapter" data-level="4.1.3" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#pdac-subtypes"><i class="fa fa-check"></i><b>4.1.3</b> PDAC Subtypes</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#descriptive-plots"><i class="fa fa-check"></i><b>4.2</b> Descriptive plots</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#boxplots"><i class="fa fa-check"></i><b>4.2.1</b> Boxplots</a></li>
<li class="chapter" data-level="4.2.2" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#scatter-plots"><i class="fa fa-check"></i><b>4.2.2</b> Scatter plots</a></li>
<li class="chapter" data-level="4.2.3" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#hierarchical-clustering"><i class="fa fa-check"></i><b>4.2.3</b> Hierarchical Clustering</a></li>
<li class="chapter" data-level="4.2.4" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#hierarchical-clustering-using-the-moffitt-transcriptomic-signature"><i class="fa fa-check"></i><b>4.2.4</b> Hierarchical clustering using the Moffitt transcriptomic signature</a></li>
<li class="chapter" data-level="4.2.5" data-path="descriptive-statistics-and-plotting.html"><a href="descriptive-statistics-and-plotting.html#heatmaps"><i class="fa fa-check"></i><b>4.2.5</b> Heatmaps</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html"><i class="fa fa-check"></i><b>5</b> RNAseq Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#rnaseq-downstream-analysis"><i class="fa fa-check"></i><b>5.1</b> RNAseq downstream analysis</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#nf-corernaseq-pipeline---installation"><i class="fa fa-check"></i><b>5.1.1</b> nf-core/rnaseq pipeline - installation</a></li>
<li class="chapter" data-level="5.1.2" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#nextflow"><i class="fa fa-check"></i><b>5.1.2</b> Nextflow</a></li>
<li class="chapter" data-level="5.1.3" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#additional-pipeline-software"><i class="fa fa-check"></i><b>5.1.3</b> Additional pipeline software</a></li>
<li class="chapter" data-level="5.1.4" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#pipeline-code"><i class="fa fa-check"></i><b>5.1.4</b> Pipeline code</a></li>
<li class="chapter" data-level="5.1.5" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#reference-genomes"><i class="fa fa-check"></i><b>5.1.5</b> Reference genomes</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#running-the-nf-corernaseq-pipeline"><i class="fa fa-check"></i><b>5.2</b> Running the nf-core/rnaseq pipeline</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#setup"><i class="fa fa-check"></i><b>5.2.1</b> Setup</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#running-the-nf-corernaseq-pipeline-1"><i class="fa fa-check"></i><b>5.3</b> Running the nf-core/rnaseq pipeline</a></li>
<li class="chapter" data-level="5.4" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#quality-control-qc-and-merged-count-files"><i class="fa fa-check"></i><b>5.4</b> Quality Control (QC) and Merged count files</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#qc-metrics"><i class="fa fa-check"></i><b>5.4.1</b> QC Metrics</a></li>
<li class="chapter" data-level="5.4.2" data-path="rnaseq-analysis.html"><a href="rnaseq-analysis.html#merged-count-files"><i class="fa fa-check"></i><b>5.4.2</b> Merged count files</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html"><i class="fa fa-check"></i><b>6</b> RNAseq Analysis - DESeq2</a>
<ul>
<li class="chapter" data-level="6.1" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#deseq2-package"><i class="fa fa-check"></i><b>6.1</b> DESEq2 package</a></li>
<li class="chapter" data-level="6.2" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#the-deseqdataset"><i class="fa fa-check"></i><b>6.2</b> The DESeqDataSet</a></li>
<li class="chapter" data-level="6.3" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#count-matrix-input"><i class="fa fa-check"></i><b>6.3</b> Count matrix input</a></li>
<li class="chapter" data-level="6.4" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#pre-filtering"><i class="fa fa-check"></i><b>6.4</b> Pre-filtering</a></li>
<li class="chapter" data-level="6.5" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#differential-expression-analysis"><i class="fa fa-check"></i><b>6.5</b> Differential expression analysis</a></li>
<li class="chapter" data-level="6.6" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#gene-annotation"><i class="fa fa-check"></i><b>6.6</b> Gene Annotation</a></li>
<li class="chapter" data-level="6.7" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#exploring-the-results"><i class="fa fa-check"></i><b>6.7</b> Exploring the results</a></li>
<li class="chapter" data-level="6.8" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#exporting-results"><i class="fa fa-check"></i><b>6.8</b> Exporting results</a></li>
<li class="chapter" data-level="6.9" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#data-transformation-and-visualisation"><i class="fa fa-check"></i><b>6.9</b> Data transformation and visualisation</a>
<ul>
<li class="chapter" data-level="6.9.1" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#pca-using-transformed-data"><i class="fa fa-check"></i><b>6.9.1</b> PCA using transformed data</a></li>
<li class="chapter" data-level="6.9.2" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#heatmaps-using-transformed-data"><i class="fa fa-check"></i><b>6.9.2</b> Heatmaps using transformed data</a></li>
</ul></li>
<li class="chapter" data-level="6.10" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#gene-set-enrichment-analysis"><i class="fa fa-check"></i><b>6.10</b> Gene Set Enrichment Analysis</a>
<ul>
<li class="chapter" data-level="6.10.1" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#kegg"><i class="fa fa-check"></i><b>6.10.1</b> KEGG</a></li>
<li class="chapter" data-level="6.10.2" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#reactome"><i class="fa fa-check"></i><b>6.10.2</b> REACTOME</a></li>
<li class="chapter" data-level="6.10.3" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#plotting-functions"><i class="fa fa-check"></i><b>6.10.3</b> Plotting functions</a></li>
<li class="chapter" data-level="6.10.4" data-path="rnaseq-analysis-deseq2.html"><a href="rnaseq-analysis-deseq2.html#extracting-enrichment-results-to-generate-heatmaps"><i class="fa fa-check"></i><b>6.10.4</b> Extracting enrichment results to generate heatmaps</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="pdac-subtypes-1.html"><a href="pdac-subtypes-1.html"><i class="fa fa-check"></i><b>7</b> PDAC Subtypes</a>
<ul>
<li class="chapter" data-level="7.1" data-path="pdac-subtypes-1.html"><a href="pdac-subtypes-1.html#towards-a-consensus-transcriptomic-classification-of-pdac"><i class="fa fa-check"></i><b>7.1</b> Towards a consensus transcriptomic classification of PDAC</a></li>
<li class="chapter" data-level="7.2" data-path="pdac-subtypes-1.html"><a href="pdac-subtypes-1.html#pdac-subtypes-and-translational-protocols"><i class="fa fa-check"></i><b>7.2</b> PDAC subtypes and translational protocols</a></li>
<li class="chapter" data-level="7.3" data-path="pdac-subtypes-1.html"><a href="pdac-subtypes-1.html#transcriptomic-profiling-of-pdac"><i class="fa fa-check"></i><b>7.3</b> Transcriptomic profiling of PDAC</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">PRECODE ESR Bioinformatics Course</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="pdac-subtypes-1" class="section level1" number="7">
<h1><span class="header-section-number">Chapter 7</span> PDAC Subtypes</h1>
<p>The identification of intrinsic subtypes using gene expression data has been successfully employed in a number of different cancer settings to define broad and potentially actionable subgroups of patients. As an exemplar, the classification of colorectal cancer by gene expression profiling has identified robust and reproducible subtypes that show promise in clinical practice. mRNA profiling of PDAC by several different groups has defined at least 4 intrinsic molecular subtypes and produced 3 major classification schemes with differing nomenclature. A comparison of these schemes highlights several important similarities and dichotomies and underlines the need to align efforts to generate a new consensus classification for PDAC that better defines patient subgroups and clinical decision-making.</p>
<p>Three major studies, in particular, have shaped debate concerning the classification of PDAC using gene expression profiles. The first of these, performed by <a href="10.1038/s41575-019-0109-y"><em>Collisson and Sadanandam et al</em></a>, used primary resected PDAC (micro-dissected to remove stromal contamination) to define 3 subtypes referred to as Exocrine-like, Classical and Quasi-mesenchymal. Genes associated with exocrine function (digestive enzyme genes), markers of epithelial adhesion and terminal differentiation (e.g. GATA6) and gain in mesenchymal function were specifically expressed in either the Exocrine-like, Classical or Quasi-mesenchymal subtypes, respectively. In addition, the Quasi-mesenchymal subtype was correlated with high tumour grade and poor patient outcomes.</p>
<p>The second major study performed by <a href="10.1038/ng.3398"><em>Moffit et al</em></a> used a supervised classification approach to informatically segregate tumour cell intrinsic gene expression signatures from “contaminating” gene expression signatures commonly associated with terminally differentiated normal pancreas (exocrine and endocrine genes signatures) and stromal cell populations (pancreatic stellate gene signatures). This analysis identified 2 major PDAC tumour cell intrinsic subtypes named Classical and Basal-like and additional tumour cell extrinsic or stromal subtypes referred to as Normal and Activated. Importantly, this study was the first to model the complex interplay between tumour cell intrinsic subtypes and specific stromal cell signals with combinations of tumour-specific and stromal subtypes associated with different patient survival.</p>
<p>The third major classification scheme proposed by <a href="https://doi.org/10.1038/nature16965"><em>Bailey et al</em></a> used primary resectable PDAC with &gt;40% cellularity to define 4 subtypes referred to as Aberrantly Differentiated Endocrine eXocrine (ADEX), Pancreatic Progenitor, Immunogenic and Squamous. These subtypes overlapped directly with the Collisson classification scheme with the exception of the Immunogenic subtype which was defined by the significant enrichment of genes associated with specific immune cell populations, including T-cells and B-cells. Although gene expression values defining the Immunogenic subtype most certainly originate from immune infiltrates resident in the tumour stroma an underlying Pancreatic Progenitor-like gene expression profile was clearly evident in tumours falling within this subtype. In addition, the Quasi-mesenchymal subtype of Collison was renamed Squamous due to the significant enrichment of several pan-squamous characteristics, including mutations in KDM6A, enrichment of the deltaNP63 isoform of p63 and a significant association with adenosquamous PDAC histology. This study also demonstrated for the first time that Squamous tumours undergo a profound epigenetic shift, with changes in DNA methylation orchestrating the down-regulation of pancreatic specific transcription factors (PDX1, GATA6, HNF1A), that control pancreatic cell fate determination, and the activation of multigene programmes regulated by deltaNTP63 and c-MYC that drive Squamous-like differentiation. Supporting a role for epigenetic dysregulation in the genesis of PDAC subtypes, the Squamous subtype was found to be enriched for mutations in COMPASS (COMplex of Proteins Associated with Set1-like) complex members KDM6A, MLL2 and MLL3 that function as chromatin modifying enzymes.</p>
<div id="towards-a-consensus-transcriptomic-classification-of-pdac" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Towards a consensus transcriptomic classification of PDAC</h2>
<p>The transcriptomic classification of PDAC by several different groups has generated a number of interesting contrasts and ultimately divided opinion. A major point of difference concerns the inclusion of the Exocrine-like/ADEX subtype as a bona fide subtype of disease. The weight of current opinion is now favouring the exclusion of this subtype on the basis that it represents normal pancreatic contamination, however, the identification of exocrine-like/ADEX gene expression in patient-derived xenographs and cell lines suggests that further study is required. A second point of contention concerns the inclusion of a separate Immunogenic subtype. Bailey et al demonstrate that the Immunogenic subtype is a complex admixture of gene expression comprising both Pancreatic Progenitor-like and immune gene expression (predominantly associated with T-cells and B-cells). The separation of the Pancreatic Progenitor signature into immune high (Immunogenic) and immune low (Pancreatic Progenitor) suggests that signals from the underlying epithelium (Immunogenic subset) may drive tumour cell immunogenicity. Recent studies, however, argue that immune infiltrates are enriched across all tumour-intrinsic subtypes and their prevalence is primarily driven by tumour cellularity of sequenced samples. In addition, these studies advocate the use of integrated classification schemes that apply both tumour-cell intrinsic and stromal subtype signatures to optimally define prognostic PDAC subtypes.</p>
<p>Despite differences in nomenclature and interpretation, a direct “side by side” comparison of the established classification schemes demonstrates considerable overlap and several common themes. In particular, strong alignment exists between the Classical/Pancreatic Progenitor and Quasi-mesenchymal/Basal-like/Squamous subtypes. Together these overlapping subtypes define 2 broad prognostic classes (referred to herein as classical and basal-like) with basal-like tumours associated with significantly poorer outcomes. These classes are delineated by the differential expression of pancreatic specific transcription factors, such as GATA6, PDX1 and HNF1A, that act to specify and maintain pancreatic identity and which are lost in basal-like tumours. Importantly, the dynamic changes in gene expression observed between the classical and basal-like classes are driven by an underlying shift in the epigenome. Multiple studies have now established that the basal-like subtype is defined by changes in DNA methylation that ultimately repress pancreatic identity and activate multigene programmes that drive Basal-like differentiation. Further, despite different approaches in modelling stromal infiltrate and the ever growing number of stromal subtypes there is a clear consensus that signals from the stroma play an important role in disease progression. An outstanding question in this regard is whether tumour-cell intrinsic subtypes contribute to the levels and/or composition of stromal (fibroblasts and immune cell) infiltrate. Additional refinement and integration of tumour-cell intrinsic and stromal subtype signatures will help to drive a greater understanding of tumour-stroma crosstalk and ultimately inform better prognostic models of disease.</p>
</div>
<div id="pdac-subtypes-and-translational-protocols" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> PDAC subtypes and translational protocols</h2>
<p>Findings from both COMPASS trial and pre-clinical models of PDAC demonstrate that basal-like tumors are less likely to respond to standard-of-care chemotherapy than classical tumors. Transcriptional profiling of pancreatic cancer patient-derived organoids has identified treatment stratification signatures (TSS) that can identify best responders to gemcitabine-based therapy, or to mFOLFIRINOX. When retrospectively applied to transcriptomic data from laser capture micro-dissected tumor tissues, obtained from either resected or advanced cases, these TSS were able to predict improved responses to mFOLFIRINOX or gemcitabine/nAb-Paclitaxel. Basal-like tumors were found to be significantly enriched in the mFOLFIRINOX non-sensitive group. As a corollary to these findings, it has recently been shown that GATA6 expression in tumors from patients with advanced disease RNA (using in situ hybridization) can discriminate classical and basal-like tumors. Importantly, the best progression-free survival in patients with advanced disease was found in GATA6 positive classical tumors given mFOLFIRINOX.</p>
<p>Detailed integrative analysis (combing both transcriptomic and genomic analyses) performed by <a href="10.1038/s41588-019-0566-9"><em>Michelle Chan-Seng-Yue et al</em></a> has demonstrated that while transcriptomic profiling can stratify PDAC into classical and basal-like subtypes the genomes of tumors falling within these same transcriptomic subtypes may exhibit considerable intra-tumour heterogeneity (ITH). Recent evidence demonstrates that both the classical and basal-like subtypes comprise at least 2 distinct subclusters that are driven by specific copy number (CN) gains in genes such as mutant KRAS and GATA6. In particular, minor and major KRAS CN imbalances can further stratify the basal-like subtype into 2 distinct subclusters, with minor KRAS CN imbalances associated with Primary disease (Stages I-III) and major KRAS CN imbalances linked to metastasis (Stage IV) and squamous-like gene expression programmes. Importantly, Stage IV tumors with Major mutant KRAS CN gains were found to be significantly more resistant to FOLFIRINOX compared to those with minor mutant KRAS CN gains. These findings are incredibly informative in that they support a model of PDAC in which ongoing genomic instability during disease progression gives rise to different molecular phenotypes.</p>
</div>
<div id="transcriptomic-profiling-of-pdac" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> Transcriptomic profiling of PDAC</h2>
<p>In this chapter, you will use your newly acquired informatic skills to analyse RNAseq data from a panel of Patient Derived Cell Lines published in <a href="https://doi.org/10.1016/j.celrep.2020.107625"><em>Brunton et al. HNF4A and GATA6 Loss Reveals Therapeutically Actionable Subtypes in Pancreatic Cancer. Cell Reports 2020</em></a></p>
<p>The RNAseq data to be analysed is available here: <a href="RNASeqData/PDCL_subtyping.RData">PDCL Subtype Data</a>.</p>
<blockquote>
<p><strong><em>Assignment - Please complete this chapter.</em></strong></p>
<p>Using the data provided:</p>
<ol style="list-style-type: decimal">
<li>Classify the PDCLs into their transcriptomic subtypes. Use the Moffitt, Collisson, Bailey and Michelle Chan-Seng-Yue classifications schemes to perform this task.</li>
<li>Compare these classification schemes - What similarities do you observe?</li>
<li>Identify biological pathways that are specifically enriched in either the Classical or Basal-like subtypes.</li>
<li>Identify a therapuetically actionable pathway signficantly enriched in either the Classical or Basal-like subtypes and define specific targets for therapy.</li>
</ol>
</blockquote>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="rnaseq-analysis-deseq2.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PRECODE_ESR_Course.pdf", "PRECODE_ESR_Course.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
